sstable/properties.go

```diff
@@ -133,9 +133,9 @@ type Properties struct {
 	// Total raw key size.
 	RawKeySize uint64 `prop:"rocksdb.raw.key.size"`
 	// Total raw rangekey key size.
-	RawRangeKeyKeySize uint64 `prop:"pebble.raw.rangekey.key.size"`
+	RawRangeKeyKeySize uint64 `prop:"pebble.raw.range-key.key.size"`
```

Comment:
These two changes are purely cosmetic, and make the range key property names consistent (i.e. hyphenated).